src/main/java/com/android/tools/build/bundletool/commands/BuildApksCommand.java

```diff
@@ -103,7 +103,7 @@ public final boolean isAnySystemMode() {
   private static final Flag<Path> OUTPUT_FILE_FLAG = Flag.path("output");
```

Comment:
This PR is not the final version, just provide a way to solve the problem. If viable, there should be more optimization.